### PR TITLE
Basic file structure for quickstart-node application

### PR DESCRIPTION
**Pull request by AvnerMaster**
The structure:
errors
services
services/monday-code - Monday code related services
routes
types
middlewares

I also added a basic README.md file

### CODE REVIEW

File: quickstart-node/package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "quickstart-node",
```
Comment (gregra81) [inline quickstart-node/package.json:2]:
```suggestion
  "name": "monday-code-quickstart",
```

File: quickstart-node/package.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "build": "npx tsc",
```
Comment (gregra81) [inline quickstart-node/package.json:6]:
```suggestion
    "build": "tsc",
```

Use typescript as a dev dependency

File: quickstart-node/src/utils/monday-code/index.ts
Comment (oricho123) [inline quickstart-node/src/utils/monday-code/index.ts:1]: im not sure if utils/monde-code fits the usages. I think it's more like monday-code services than utils. What do you think?

Comment (AvnerMaster) [reply quickstart-node/src/utils/monday-code/index.ts:1]: done

**Review by oricho123 — APPROVED** [review]
LGTM 💯

**Pull request merged**